tests/unittests/Foundation/ReferenceFoundation/TestNSArray.mm

```diff
@@ -150,13 +150,19 @@ static int objectIndexInArray(NSArray* array, int value, int startingFrom, int l
     //        NSBinarySearchingOptions* searchForBoth = NSBinarySearchingFirstEqual | NSBinarySearchingLastEqual
     //        auto _ = objectIndexInArray(array, 9, 0, 13, searchForBoth)
 
-    auto notFound = objectIndexInArray(array, 11, 0, 13);
-    ASSERT_EQ_MSG(notFound, NSNotFound, @"NSArray return NSNotFound if object is not found.");
+    ASSERT_EQ_MSG(objectIndexInArray(array, 11, 0, array.count),
```

Comment:
Curious why this was changed. In general I agree that using magic numbers is bad but this is also changing the code path slightly. With the change we are testing the whole array, whereas before we were testing a subset. I think both are good test cases and we should probably include both (especially for the below case dealing with index of 9).